File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-python/codex-sh/quicksort.py
--- 
+++ 
@@ -4,5 +4,6 @@
 
     pivot = arr[0]
     lesser = quicksort([x for x in arr[1:] if x < pivot])
-    greater = quicksort([x for x in arr[1:] if x > pivot])
+    greater = quicksort([x for x in arr[1:] if x >= pivot])
+
     return lesser + [pivot] + greater

File: proj/code-repair/mycode/clean_dataset/patch-single-line/quixbugs-java/gpt-neox/quicksort.java
--- 
+++ 
@@ -10,7 +10,7 @@
     for (Integer x : arr.subList(1, arr.size())) {
         if (x < pivot) {
             lesser.add(x);
-        } else if (x > pivot) {
+        } else {
             greater.add(x);
         }
     }
